traffic_ops/testing/api/v4/regions_test.go

```diff
@@ -175,15 +226,90 @@ func UpdateTestRegions(t *testing.T) {
 	if err != nil {
 		t.Errorf("cannot UPDATE Region by id: %v - %v", err, alert)
 	}
-
 }
 
-func GetTestRegions(t *testing.T) {
-	for _, region := range testData.Regions {
-		resp, _, err := TOSession.GetRegionByName(region.Name, nil)
-		if err != nil {
-			t.Errorf("cannot GET Region by region: %v - %v", err, resp)
+func VerifyPaginationSupportRegion(t *testing.T) {
+
+	qparams := url.Values{}
+	qparams.Set("orderby", "id")
+	regions, _, err := TOSession.GetRegions(qparams, nil)
+	if err != nil {
+		t.Fatalf("cannot GET Regions: %v", err)
+	}
+
+	if len(regions) > 0 {
+		qparams = url.Values{}
+		qparams.Set("orderby", "id")
+		qparams.Set("limit", "1")
+		regionsWithLimit, _, err := TOSession.GetRegions(qparams, nil)
+		if err == nil {
+			if !reflect.DeepEqual(regions[:1], regionsWithLimit) {
+				t.Error("expected GET Regions with limit = 1 to return first result")
+			}
+		} else {
+			t.Error("Error in getting regions by limit")
 		}
+
+		if len(regions) > 1 {
```

Comment:
The `regions` slice in here is sliced to `1:2` which will segfault if it can't get index `1` (or, equivalently, `:2`) - which would mean they need to have a length of at least 2

Reply:
yes, if the condition falls `if len(regions) > 1` it should be greater than or equal to 2. It won't segfault.

Reply:
oh,. you're right, of course. My mistake